components/brave_wallet/browser/swap_service_unittest.cc

```diff
@@ -556,6 +556,20 @@ TEST_F(SwapServiceUnitTest, GetPriceQuoteURL) {
             "affiliateAddress=0xbd9420A98a7Bd6B89765e5715e169481602D9c3d&"
             "skipValidation=true");
 
+  url = swap_service_->GetPriceQuoteURL(params.Clone(),
```

Comment:
can we make these `GetPriceQuoteURL` and `GetTransactionPayloadURL` checks into a loop and leverage `base::StringPrintf`? url parameters are basically the same, if there are differences `base::StringPrintf` can still handle that.

Reply:
And don't forget to use `SCOPED_TRACE` in the loop so we can easily tell which case went wrong if test failed

Reply:
@darkdh Fixed here: https://github.com/brave/brave-core/pull/19672/commits/5747a18326b72ece5daa19ec5326bcef3e0f29c0

Separated out the RFQ and non-RFQ cases for clarity, since the URLs are a bit different.

Reply:
that's fine. looks cleaner now.